src/Views/IframeView.php

```diff
@@ -273,8 +278,8 @@ class="js-give-embed-form-modal-opener"
 				data-form-id="%1$s"%3$s>%2$s</button>
 			</div>',
             $this->uniqueId,
-            $this->buttonTitle,
-            $this->buttonColor ? " style=\"background-color: {$this->buttonColor}\"" : ''
+            $buttonTitle,
+            $this->buttonColor ? " style=\"background-color: {$buttonColor}\"" : ''
```

Comment:
Can we actually use late escaping here? That way the previous variables can not be altered and it is more clear when looking at the sprintf

Reply:
Resolved in 1b785dd4e74939d345c573eec409f13e1420e736